.github/workflows/lint.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         id: lint
         with:
           version: latest
-          args: '-c .github/.golangci.yml --out-format=colored-line-number'
+          args: '-c .github/.golangci.yml --out-format=colored-line-number --timeout 2m0s'
```

Comment:
Why is this required though?

Reply:
The default appears too little for the linter. Lots of PRs for various projects increase this.

@paul-rodriguez It looks like this PR from @barrettj12 will set it to 5m using the [run configuration](https://golangci-lint.run/usage/configuration/#run-configuration) key ```timeout``` which is directly set in the config file ```.github/.golangci.yml```.

Maybe we should do the same, which means no conflicting attempts setting timeouts. 5m used by Jordan seems future safe?